doc/hacking.rst

```diff
@@ -39,6 +39,18 @@ Assignment above) needs to be licensed under the `MIT license`_ that testtools
 uses, so that testtools can ship it.
 
 
+Installing
```

Comment:
I don't think installing makes sense *here* in the docs: this is docs on hacking, and the 'testing' section below covers this.

Secondly, for users, I think we want to say 'pip install testtools' - anyone installing from source should be coming from a position of knowledge after all...